src/main/java/com/yahoo/identity/services/account/Account.java

```diff
@@ -31,4 +32,13 @@
 
     @Nonnull
     String getDescription();
+
+    @Nonnull
+    Instant getBlockUntilTime();
+
+    @Nonnull
+    int getConsecutiveFails();
+
+    @Nonnull
+    Boolean verify(@Nonnull String password);
```

Comment:
Why the return value is defined as `Boolean`, not `boolean`?

Reply:
There is no benefit from `Boolean`. Changed to the primitive type now.